conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowEteResource.java

```diff
@@ -172,6 +173,14 @@ public SimpleEnum enumHeader(AuthHeader _authHeader, SimpleEnum headerParameter)
         return headerParameter;
     }
 
+    @Override
+    public String jsonErrorsHeader(AuthHeader authHeader, String headerParameter) {
+        if (headerParameter.equals("JSON") || headerParameter.equals("TOSTRING")) {
```

Comment:
nit: let's just throw if the header is set? (or even just always throw?)
```suggestion
        if (!headerParameter.isEmpty()) {
```

Reply:
Updated to always throw. Initially I thought I'd test that we're able to get the standard return type from this endpoint, but this can be handled with other tests and doesn't relate to this PR.